app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>TaskList</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
```

Comment:
You really should fill in this file to provide a template for the content on each page.  That would make it easier to navigate between pages etc.  

Consider moving some of the content from `index.html.erb` to this page.